deployment.rb

```diff
@@ -187,6 +187,18 @@ def hourofcode_hostname
     canonical_hostname('hourofcode.com')
   end
 
+  def circle_run_identifier
+    ENV['CIRCLE_BUILD_NUM'] ? "CIRCLE-BUILD-#{ENV['CIRCLE_BUILD_NUM']}-#{ENV['CIRCLE_NODE_INDEX']}" : nil
```

Comment:
@islemaster hm, what do you think about that? I think how it is right now might be good, since N > 2 containers will split the tests out and there won't be any overlapping test files (so theoretically the test status page would then give an overview of all of the tests across all containers)